instance/templates/instance/ansible/vars.yml

```diff
@@ -83,5 +83,5 @@ EDXAPP_WORKER_DEFAULT_STOPWAITSECS: 1200
 COMMON_ENABLE_NEWRELIC: {% if newrelic_license_key %}true{% else %}false{% endif %}
 COMMON_ENABLE_NEWRELIC_APP: {% if newrelic_license_key %}true{% else %}false{% endif %}
 COMMON_ENVIRONMENT: opencraft
-COMMON_DEPLOYMENT: '{{ instance.sub_domain }}'
+COMMON_DEPLOYMENT: '{{ instance.internal_lms_domain }}'
```

Comment:
@itsjeyd It doesn't matter, as long as it's unique for each instance. As far as I know, it is only used for displaying the name of the app in New Relic dashboard.